source/Irrlicht/COpenGLDriver.h

```diff
@@ -470,7 +471,8 @@ class COpenGLDriver final : public CNullDriver, public COpenGLFeatureMap, public
 				default:
 				{
 					GLint res = GL_FALSE;
-					glGeneral.pglGetInternalformativ(GL_TEXTURE_2D, getSizedOpenGLFormatFromOurFormat(_fmt), GL_COLOR_RENDERABLE, sizeof(res), &res);
+					//this is troublesome, how to obtain context reference here?
+					//functionTable.glGeneral.pglGetInternalformativ(GL_TEXTURE_2D, getSizedOpenGLFormatFromOurFormat(_fmt), GL_COLOR_RENDERABLE, sizeof(res), &res);

```

Comment:
its not a static function, so it should be fine, the only problem is the `const`.... @Crisspl any advice?

Maybe we could make the `operator()` on the function pointer wrapper class a `const` operator?